packages/react-components/src/components/HorizontalGallery.tsx

```diff
@@ -76,9 +76,12 @@ export const HorizontalGallery = (props: HorizontalGalleryProps): JSX.Element =>
   const clippedPage = firstIndexOfCurrentPage < numberOfChildren - 1 ? page : lastPage;
 
   const childrenOnCurrentPage = useMemo(() => {
-    return indexesArray[clippedPage].map((index) => {
-      return React.Children.toArray(children)[index];
-    });
+    if (indexesArray[0] !== undefined) {
```

Comment:
I'm curious what situation would the first index be undefined

Reply:
When there is no overflow gallery tiles this will be undefined and will throw an error so we only want to be able to call the map if there is a tile here and the overflow is present